packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -146,20 +146,23 @@ export const TooltipPlugin2 = ({
 
     let _plot = plot;
     let _isHovering = isHovering;
+    let _someSeriesIdx = false;
     let _isPinned = isPinned;
     let _style = style;
 
+    const updHovering = () => {
+      _isHovering = closestSeriesIdx != null || hoverMode === TooltipHoverMode.xAll && _someSeriesIdx;
+    };
+
     let offsetX = 0;
     let offsetY = 0;
 
-    let htmlEl = document.documentElement;
-    let winWidth = htmlEl.clientWidth - 16;
-    let winHeight = htmlEl.clientHeight - 16;
-
-    window.addEventListener('resize', (e) => {
-      winWidth = htmlEl.clientWidth - 16;
-      winHeight = htmlEl.clientHeight - 16;
-    });
+    let containRect = {
+      lft: 0,
```

Comment:
This is definitely nitpicking but feels like clarity is being sacrificed for conciseness in these variable names.

Reply:
maybe.

let's revisit this later. for me alignment makes things much more readable than having whole words spelled out. part of the reason i hate prettier, it completely screws up extra spaces added to make the logic between very similar blocks easily recognizable.

```
left: 0,
top: 0,
bottom: screen.height,
right: screen.width,
```

this might be okay-ish, until you need to start using "top" and "bottom" more than once in the same logical expression in a single line.

Reply:
i'm torn on these --but honestly, my eyes can figure out what is happening with the 4 equivalent things when they have the same length.  If the linter allowed aligning by `=` but alas.

no strong feeling either way